src/stash-list/tab.vue

```diff
@@ -48,6 +49,10 @@ export default defineComponent({
         targetWindow(): number | undefined {
             return this.model().tabs.targetWindow.value;
         },
+        isLoading(): boolean {
+            return this.tab.status === 'loading' && 
+                this.tab.windowId === this.targetWindow;
+        },
```

Comment:
I don't think you need the `targetWindow` part of this check; tabs can be loading regardless of whether they're in the target window or not.

(TBH I'm not sure it's even needed for `isActive`; it may be detritus from a prior refactoring, when `tab.vue` and `bookmark.vue` used to be the same component. I don't think we ever use `tab.vue` for tabs outside of the target window anyway.)

Reply:
My thinking was that we didn't want to show a loading indicator for a tab that's not loading in the current window. Maybe that already wouldn't happen? I will change it.

Reply:
Yeah your thinking makes sense. But I don't think it will happen anyway because the only tabs we show in the first place are tabs in the current window. So this check should be redundant.

Reply:
Fair enough!